HeterogeneousCore/AlpakaUtilities/interface/radixSort.h

```diff
@@ -0,0 +1,251 @@
+#ifndef AlpakaCore_radixSort_h
+#define AlpakaCore_radixSort_h
+
+#include <cstdint>
+#include <type_traits>
+
+#include "HeterogeneousCore/AlpakaInterface/interface/config.h"
```

Comment:
Can this be replaced as
```suggestion
#include <alpaka/alpaka.hpp>
```
?